src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,53 @@
+package coffee.order;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class CoffeeOrderBoard {
+
+    private List<Order> orders;
+    private Map<Integer, Order> orderMap;
+
+    public CoffeeOrderBoard() {
+        orders = new ArrayList<>();
```

Comment:
as we discuss on lesson ArrayList is not optimal it is better to use queue (linkedList)